core/domain/exp_services.py

```diff
@@ -407,6 +407,10 @@ def apply_change_list(exploration_id, change_list):
                         state_domain.WrittenTranslations.from_dict(
                             change.new_value))
                     state.update_written_translations(written_translations)
+            elif change.cmd == exp_domain.CMD_ADD_TRANSLATION:
```

Comment:
tests? (There should be some change in exp_services_test...)

Reply:
@seanlip I've already added the tests, you can find one just below this file.